.storybook/preview.tsx

```diff
@@ -14,5 +15,9 @@ export const parameters = {
 };
 
 export const decorators = [
-  renderStory => <ThemeProvider>{renderStory()}</ThemeProvider>,
+  renderStory => (
```

Comment:
The static class prefix by default is `mantine`. However, when using an emotion cache, it changes to the key of the emotion cache. By default our storybook previews didn't have a cache set, so the prefixes were different between storybook and the metabase app ('css' and 'emotion') respectively. This change brings those back in line.